service/endpoint/internal/src/main/java/org/eclipse/kapua/service/endpoint/internal/EndpointInfoServiceImpl.java

```diff
@@ -167,7 +180,11 @@ public EndpointInfo find(KapuaId scopeId, KapuaId endpointInfoId)
 
         //
         // Check Access
-        AUTHORIZATION_SERVICE.checkPermission(PERMISSION_FACTORY.newPermission(EndpointInfoDomains.ENDPOINT_INFO_DOMAIN, Actions.read, scopeId));
+        KapuaId scopeIdPermission = entityManagerSession.doAction(em -> EndpointInfoDAO.find(em, scopeId, endpointInfoId))
```

Comment:
If `entityManagerSession.doAction(em -> EndpointInfoDAO.find(em, scopeId, endpointInfoId))` returns null, we have a `NullPointerException` here. Let's test this first.